db/migrate/20170412000117_convert_private_flag_to_visibility_in_predictions.rb

```diff
@@ -0,0 +1,19 @@
+class ConvertPrivateFlagToVisibilityInPredictions < ActiveRecord::Migration
+  def up
+    sql = <<-EOS
+    UPDATE predictions
+    SET visibility = 1
+    WHERE private = true
+    EOS
+    ActiveRecord::Base.connection.execute sql
```

Comment:
I disagree. Raw SQL is much better in migrations, where you can guarantee the current state of the database structure. I believe your suggested change would fail if I ran the migrations from scratch, because the `private` attribute no longer exists on the `Prediction` model. I could, alternatively, override the model in the migration, but I think that's more trouble than it's worth.